### ISSUE TITLE
Building the documentation in the repo

### ISSUE THREAD

**Issue by JulieSchramm** [NOAA-EMC/regional_workflow#40]
If you would like to build the html and pdf version of the documentation located in doc/user_guilde, you (or your sys admin) will need to install Sphinx on your desktop/latop from:

http://www.sphinx-doc.org/en/master/usage/installation.html

You don't need to run quickstart, since those files have already been created and are in the doc/user_guide directory.  Once installed, you can cd doc/user_guide and type 'make html'.

**Comment by JacobCarley-NOAA** [comment]
Hi @JulieSchramm 

I tried building the documentation following the instructions to install sphinx on my macbook but keep getting the following error:

>(base) jrc:user_guide jrcarley$ make html
>Running Sphinx v2.1.2
>
>Extension error:
>Could not import extension sphinxcontrib.bibtex (exception: No module named 'sphinxcontrib.bibtex')
>make: *** [html] Error 2

I tired installing the missing extension noted above (via conda install) and still get the same error.  Has anyone else tried to run this on their system? @BinLiu-NOAA @BenjaminBlake-NOAA ?

**Comment by JulieSchramm** [comment]
Hi Jacob,

Did you install the Sphinx bibtex from here?

https://sphinxcontrib-bibtex.readthedocs.io/en/latest/quickstart.html#installation


I had a sys-admin install it on my Mac, since they don't give us permission
to install software.  If you are still having problems, here is a
workaround:


cd doc/user_guide/source

vi conf.py


Comment out the line with the bibtex in


extensions = [

    'sphinx.ext.autodoc',

    'sphinx.ext.doctest',

    'sphinx.ext.intersphinx',

    'sphinx.ext.todo',

    'sphinx.ext.coverage',

    'sphinx.ext.mathjax',

    'sphinx.ext.ifconfig',

    'sphinx.ext.viewcode',

    'sphinx.ext.githubpages',

    'sphinx.ext.napoleon',

    'sphinxcontrib.bibtex'

]

Let me know if this doesn't work.

Thanks,

Julie

**Comment by JacobCarley-NOAA** [comment]
@JulieSchramm I did try that.  Interestingly I can import it from the command line running python, but I use python3.  After some additional Googling I edited the Makefile and did the following which has seemed to work:

Replaced

`SPHINXBUILD   = sphinx-build`

with

`SPHINXBUILD   = python3 -m sphinx`

And the html pages have now appeared!

Relevant stackoverflow post: 

https://stackoverflow.com/questions/50800270/how-can-i-get-sphinx-to-use-python-3-as-an-interpreter
